lib/assembly/cache.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Assembly.Cache do
```

Comment:
So here is where we want to ask how async we want this to be...

If we do it async (like this setup) when we cold boot with 30 builds 10 components each, we are looking at (max) 301 queue messages for quantity requests. This doesn't hurt anything, but it's a lot of un-needed messages.

If we do it fully sync any inventory goes down, it will basically crash this service too.

Personally, I think that `execute/1` should be sync. This will avoid the huge amount of calls on startup, and will only crash if both services are down at the same time (we probably have bigger problems.) I think everything else seems good.